README.md

```diff
@@ -1,2 +1,41 @@
 # gold-starter
-Wrapper for the gold project to provide endpoints for experimentation.
+Wrapper for the [gold](https://github.com/nfl/gold) project to provide endpoints for experimentation.
+
+## Architecture
+* _REST Controllers_: The project employs Spring Boot and Spring Web to expose _gold_ services
+* _Services_: _gold-starter_ simply autowires all Spring components declared in _gold_
+  and uses _gold_ Facade: GraphQLSchemaService.executeQuery(...) and GraphQLInstanceService.executeQuery(...).
+* _Persistence Layer_: In memory hash maps are used to get the project up and running in no time.
+
+## Features
+For the complete documentation of supported features, queries, mutations, etc please refer
+to [gold documentation](https://github.com/nfl/gold).
+This project simply exposes _gold_via REST.
+So, this document only mentions some default and tweaks it applies before calling _gold_.
+
+## Tweaks / defaults
+To make experimentation simpler, _gold-starter_ applies a few defaults before calling _gold_ services:
+* _Security_: REST controllers automatically grant all _gold_ permissions to all REST calls.
+* _Namespaces_: All schemas are in _GOLD_STARTER_SCHEMA_NAMESPACE_.
+  All instances are in _GOLD_STARTER_INSTANCE_NAMESPACE_.
+  No need to worry about namespaces or to specify them in any REST calls.
+* _Nesting depth_ of queries is limited to 5 levels.
+
+## REST endpoint to manage schemas: `/gold/definitions?query={query_contents}&schemaName={dynamic_domain_name}`
+Since this project simply delegates all processing to _gold_, it supports the same set of operations to
+view or modify a schema definition declared in [gold documentation](https://github.com/nfl/gold). 
+For a quick start:
+* POST a [schema of a Pet](examples/pet/schema_with_lizard.txt) to the *schema definition endpoint*.
+* GET all schema definitions using [this query](examples/pet/schema_with_lizard.txt) to the same endpoint.
+
+## REST endpoint to manage instances: `/gold/instances/{schemaName}?query={query_contents}`
+Assuming you have defined a Pet schema as suggested in the section above,
+you can populate the in memory repository with little effort:
+* POST some pets ([katie_dog.txt](examples/pet/katie_dog.txt),
```

Comment:
Show the exact URL for this ... i.e. /gold/instances/Pet?query=`contents of katie_dog.txt.

As an aside, I'll be adding a demo "script/notes" going through how one would demo the examples.